kiwi/storage/subformat/__init__.py

```diff
@@ -85,7 +86,7 @@ def __new__(self, name, xml_state, root_dir, target_dir):
             return DiskFormatGce(
                 xml_state, root_dir, target_dir, custom_args
             )
-        elif name == 'vmdk':
+        elif name == 'vmdk' or name == 'ova' or name == 'ovf':
```

Comment:
I think it doesn't have much sense to keep 'ovf' format as a valid option. Allowing format="ova" and format="ovf" knowing the result will be identical is confusing IMHO.

In fact this issue was already here, in advance of this PR. The schema allows to set 'ova' or 'ovf' as format while there is or there was no code to handle it, leading to a late failure during the create step. I think we could consider removing the 'ovf' format as a valid one in the schema, but well, I could also live leaving it as is, this is just a mere opinion.

@schaefi what do you think?

Reply:
I agree it's confusing to keep 'ovf' to be handled in the code.

Let's delete this and keep only:

```python
elif name == 'vmdk' or name == 'ova':
```

Once merged let us create a follow up pull request which deletes the ovf value from the format attribute to get an early error condition

Thoughts ?